configure

```diff
@@ -1518,10 +1518,10 @@ check_thread_libs() {
 
     # additional search paths by the compiler
     local cc_lib_search_paths=""
-
     case "$cxx_compiler_type" in
 	gnu|clang)
 	    cc_lib_search_paths=$(${CXX} -print-search-dirs | sed -n -e '/libraries:/s/libraries: *=//p')
+	    cc_lib_search_paths=$cc_lib_search_paths$(${CXX} -Xlinker -v 2>&1 | sed -n '/Library search paths:/,/Framework search paths:/p' | sed '/Library search paths:/d; /Framework search paths:/d' | sed -e ':a' -e 'N' -e '$!ba' -e 's/\n/ /g' | tr -s '[:blank:]' ':')
```

Comment:
I would recommend to replace `2>&1` by `2>/dev/null`, because we don't need to parse the error output.

Reply:
I don't think that works.  Commands with that were failing and Dylan told me that the entire output is error so do need tp parse it.

Reply:
Ok, then we must leave it.